### DISCUSSION TITLE
Quartus Prime Lite with rosetta 2

### DISCUSSION THREAD

**Discussion by Pedroo64** [lima-vm/lima#1248] [Q&A] [answered]
Hello,

I've been trying to install and run Quartus Prime Lite which is a x64 app on the m1 pro using lima-vm with rosetta 2 using vz as vmType.

When I try to install Quartus Prime Lite by launching .run it crashes and gives a segmentation fault error.

Is there a way to figure out what causes the crash ?

I've tried to install Quartus Prime Lite with docker using docker x64 emulation. The install process doesn't crash and the app runs. The problem is that synthesis phase loops forever...

Thank you

Pedro

**Comment by AkihiroSuda** [comment, marked as answer]
Rosetta is Apple's proprietary component of macOS, so this should be better reported to Apple
